### PR TITLE
Update cartesian_coordinates.astro

### PR DESCRIPTION
**Pull request by kelliehalloran**

### CODE REVIEW

File: src/pages/sta/cartesian_coordinates.astro [hunk 1]
```diff
@@ -7,15 +7,297 @@ import BlueText from "../../components/BlueText.astro"
...
+        </CalloutCard>
+    </CalloutContainer>
+</SubSection>
 
 </Section>
 
 <SubSection title="Vector Bases">
 
-<BlueText>Insert the same content here as from the TAM212 Reference Pages (Vectors and Bases - Vector Bases)</BlueText>
+<SubSection title="Vectors bases">
```
Comment (mwest1066) [inline src/pages/sta/cartesian_coordinates.astro:113]: I think we have an extra `SubSection` here? We're already inside a very similar `SubSection`.

File: src/pages/sta/cartesian_coordinates.astro [hunk 2]
```diff
@@ -7,15 +7,297 @@ import BlueText from "../../components/BlueText.astro"
...
+              <p>
+                Writing them as a list assumes that we know which
+                coordinate system we are using, and which order the
+                coordinates are in. We will typically write the coordinate
+                values explicitly to avoid confusion. This also applies to
+                <a href="rvv.html#rvv-ic">vector components</a>.
+              </p>
+        </CalloutCard>
+
+        <CalloutCard>
```
Comment (mwest1066) [inline src/pages/sta/cartesian_coordinates.astro:90]: I know that this is copied from Dynamics, which is generally fine. However, I think we probably don't want to copy the "Did you know?" callout cards. These are more "fun facts" and I think it's going to be a bit weird to have them show up in multiple courses?

File: src/pages/sta/cartesian_coordinates.astro [hunk 3]
```diff
@@ -7,15 +7,297 @@ import BlueText from "../../components/BlueText.astro"
...
+          coordinates</a>, we can write the vector components
+          \(3\hat\imath + 2\hat\jmath\) as the ordered list \((3, 2)\)
+          if we know which basis we are using. Because we often will
+          be using several bases simultaneously, we will generally
+          write the components explicitly in the \(3\hat\imath +
+          2\hat\jmath\) form.
+        </p>
+      </CalloutCard>
+
+      <CalloutCard>
```
Comment (mwest1066) [inline src/pages/sta/cartesian_coordinates.astro:179]: Same here, I think we shouldn't copy this "Did you know?" card.

File: src/pages/sta/cartesian_coordinates.astro [hunk 4]
```diff
@@ -7,15 +7,297 @@ import BlueText from "../../components/BlueText.astro"
...
+    </p>
+    <p>
+      It will always be true, however, that <InlineEquation equation="\\|\\vec{c}\\| \\le \\|\\vec{a}\\| + \\|\\vec{b}\\|" />. This fact is known as the
+      <em>triangle inequality</em>, for reasons that should be
+      obvious.
+    </p>
+</Warning>
+
+<CalloutContainer slot="cards">
+  <CalloutCard>
```
Comment (mwest1066) [inline src/pages/sta/cartesian_coordinates.astro:273]: And another "Did you know?" that I think we shouldn't copy.

**Comment by jcrayb** [comment]
The changes from this branch have somehow already been deployed, so I'll close it for now (minus the "Did you know?" cards).

**Pull request closed without merging**